src/components/navigation/problem-tab-content.sass

```
@@ -86,5 +88,8 @@
           background-color: $learninglog-green-light-6
     .solutions-label
       margin: 0 7px 0 3px
+  .problems-panel
```

```suggestion
  .problem-panel
```
as pointed out elsewhere, I think there's only a single problem here.